packages/server/database/migrations/20161125141100-addOrgs.js

```diff
@@ -119,21 +118,23 @@ exports.up = async (r) => {
       users[userId].trialOrg = orgId
     }
   }
-  const orgIds = Object.keys(orgs)
-  const stripeCustomers = await Promise.all(
-    orgIds.map((orgId) => stripe.customers.create({metadata: {orgId}}))
-  )
-  const subscriptions = await Promise.all(
-    stripeCustomers.map((customer) => {
-      return stripe.subscriptions.create({
-        customer: customer.id,
-        metadata: customer.metadata,
-        plan: 'parabol-pro-monthly',
-        quantity: Object.keys(orgs[customer.metadata.orgId].orgUserMap).length,
-        trial_period_days: TRIAL_PERIOD_DAYS
-      })
-    })
-  )
+  // turned into a noop to remove dependency on stripe
+  const subscriptions = []
+  // const orgIds = Object.keys(orgs)
```

Comment:
deleted the stripe helper, which this depends on, so had to remove any logic using it.
in the future, we shouldn't do stripe things in migrations, but this was 6 years ago